src/components/timer/TimerController.cpp

```diff
@@ -4,32 +4,29 @@
 
 #include "components/timer/TimerController.h"
 #include "systemtask/SystemTask.h"
-#include "app_timer.h"
 #include "task.h"
 
 using namespace Pinetime::Controllers;
 
-
-APP_TIMER_DEF(timerAppTimer);
-
 namespace {
-  void TimerEnd(void* p_context) {
-    auto* controller = static_cast<Pinetime::Controllers::TimerController*> (p_context);
-    if(controller != nullptr)
-      controller->OnTimerEnd();
+  void TimerEnd(TimerHandle_t xTimer) {
+    auto controller = static_cast<Pinetime::Controllers::TimerController*>(pvTimerGetTimerID(xTimer));
+    controller->OnTimerEnd();
   }
 }
 
-
-void TimerController::Init() {
-  app_timer_create(&timerAppTimer, APP_TIMER_MODE_SINGLE_SHOT, TimerEnd);
+void TimerController::Init(System::SystemTask* systemTask) {
+  this->systemTask = systemTask;
+  timerAppTimer = xTimerCreate("timerAppTm", 1, pdFALSE, this, TimerEnd);
 }
 
 void TimerController::StartTimer(uint32_t duration) {
-  app_timer_stop(timerAppTimer);
+  xTimerStop(timerAppTimer,0);
```

Comment:
Please use clang-format to correctly format all modified files.

Reply:
Oops missed that space.  I'll try to reformat each commit.

Reply:
Done now